plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/WapitiReader.java

```diff
@@ -106,47 +106,17 @@ public TestSuiteResults parse(ResultFile resultFile) throws Exception {
     }
 
     // Parse the CWE # out of the references included with the vuln
-    private int getCWE(Node vuln) {
-        int cwe = -1;
+    private CweNumber getCWE(Node vuln) {
+        CweNumber cwe = CweNumber.DONTCARE;
         Node refs = getNamedChild("references", vuln);
         List<Node> references = getNamedChildren("reference", refs);
         for (Node ref : references) {
             String title = getNamedChild("title", ref).getTextContent();
             if (title.startsWith("CWE-")) {
                 String cweNum = title.substring("CWE-".length(), title.indexOf(":"));
-                cwe = cweLookup(cweNum);
+                cwe = CweNumber.lookup(cweNum);
```

Comment:
Can we restore the old cweLookup so we can still get the warning "WARNING: Wapiti-Unmapped CWE number: " + cwe);"?

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379